### ISSUE TITLE
conda update linking packages error

### ISSUE THREAD

**Issue by yusufazishty** [conda/conda#2722]
I do : conda update --all
than after download all packages:
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
An unexpected error has occurred, please consider sending the            |  30%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "C:\Anaconda3\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Anaconda3\lib\site-packages\conda\cli\main.py", line 120, in main
    args_func(args, p)
  File "C:\Anaconda3\lib\site-packages\conda\cli\main.py", line 127, in args_func
    args.func(args, p)
  File "C:\Anaconda3\lib\site-packages\conda\cli\main_update.py", line 64, in execute
    install(args, parser, 'update')
  File "C:\Anaconda3\lib\site-packages\conda\cli\install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Anaconda3\lib\site-packages\conda\plan.py", line 566, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Anaconda3\lib\site-packages\conda\instructions.py", line 137, in execute_instructions
    cmd(state, arg)
  File "C:\Anaconda3\lib\site-packages\conda\instructions.py", line 80, in LINK_CMD
    link(state['prefix'], dist, lt, index=state['index'], shortcuts=shortcuts)
  File "C:\Anaconda3\lib\site-packages\conda\install.py", line 1040, in link
    os.makedirs(dst_dir)
  File "C:\Anaconda3\lib\os.py", line 237, in makedirs
    mkdir(name, mode)
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\Anaconda3\\Lib/site-packages/MarkupSafe-0.23-py3.4-win-amd64.egg-info'

I don't understand what gone wrong. I updated it once before and all just fine

**Comment by kalefranz** [comment]
Oh another FileExistsError.  Probably a duplicate report of #2673.

**Comment by kalefranz** [comment]
Also duplicate of #2714

**Comment by stippingerm** [comment]
I have got a similar error (Anaconda4, environment py27 with python 2.7, numpy already installed):

```
[py27] C:\Windows\system32>conda install pandas
Fetching package metadata .......
Solving package specifications .............

Package plan for installation in environment C:\Program Files\Anaconda3\envs\py2
7:

The following NEW packages will be INSTALLED:

    pandas:          0.18.1-np111py27_0
    python-dateutil: 2.5.3-py27_0
    pytz:            2016.4-py27_0
    six:             1.10.0-py27_0

Proceed ([y]/n)?

Extracting packages ...
An unexpected error has occurred, please consider sending the            |  75%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Program Files\Anaconda3\Scripts\conda-script.py", line 5, in <module>

    sys.exit(main())
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\cli\main.py", line 12
0, in main
    args_func(args, p)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\cli\main.py", line 12
7, in args_func
    args.func(args, p)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\cli\main_install.py",
 line 69, in execute
    install(args, parser, 'install')
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\cli\install.py", line
 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\plan.py", line 566, i
n execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\instructions.py", lin
e 137, in execute_instructions
    cmd(state, arg)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\instructions.py", lin
e 60, in EXTRACT_CMD
    extract(arg)
  File "C:\Program Files\Anaconda3\lib\site-packages\conda\install.py", line 828
, in extract
    os.rename(temp_path, path)
PermissionError: [WinError 5] Access is denied: 'C:\\Program Files\\Anaconda3\\p
kgs\\pandas-0.18.1-np111py27_0.tmp' -> 'C:\\Program Files\\Anaconda3\\pkgs\\pand
as-0.18.1-np111py27_0'

[py27] C:\Windows\system32>conda info
Current conda install:

             platform : win-64
        conda version : 4.1.1
  conda-build version : 1.20.0
       python version : 3.5.1.final.0
     requests version : 2.9.1
     root environment : C:\Program Files\Anaconda3  (read only)
  default environment : C:\Program Files\Anaconda3\envs\py27
     envs directories : C:\Users\User\.conda\envs
                        C:\Users\User\envs
                        C:\Program Files\Anaconda3\envs
        package cache : C:\Users\User\.conda\envs\.pkgs
                        C:\Users\User\envs\.pkgs
                        C:\Program Files\Anaconda3\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : None
    is foreign system : False
```

**Comment by kalefranz** [comment]
fixed with #2717
